ExilenceNextApp/public/i18n/en/common.json

```diff
@@ -49,13 +51,17 @@
     "session_id_icon_title": "Help finding session ID",
     "account_icon_title": "My account",
     "net_worth_export": "Export data",
+    "these_terms_inline": "these terms",
     "update_available": "Version {{version}} available"
   },
   "link": {
     "restart_now": "Restart now"
   },
   "body": {
     "clear_snapshots": "Are you sure you want to clear all snapshots for the current profile?",
-    "remove_profile": "Are you sure you want to remove your profile?"
+    "remove_profile": "Are you sure you want to remove your profile?",
+    "ga_consent_main_text": "We use Google Analytics in Exilence Next. This means we track how you interact with the app, in order to improve it further.",
+    "ga_consent_sub_text": "To clarify, we don't track anything related to your Path of Exile account in Google Analytics, and therefor can't identify you as a user.",
```

Comment:
Remove  _therefor_ so the message becomes _To clarify, we don't track anything related to your Path of Exile account in Google Analytics, and can't identify you as a user._

Reply:
Fixed